package.json

```diff
@@ -102,7 +102,7 @@
   },
   "volta": {
     "node": "18.20.4",
-    "pnpm": "9.12.2"
+    "pnpm": "10.4.0"
```

Comment:
Before merging, we should run `pnpm approve-builds` so that the root `package.json` is updated to allow lefthook, etc. to run